addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestRedirectUrlSupplier.java

```diff
@@ -37,11 +37,11 @@
 
     private final Logger logger = LoggerFactory.getLogger(NestRedirectUrlSupplier.class);
 
-    private String cachedUrl = "";
+    protected String cachedUrl = "";
 
```

Comment:
Why? 🙂

Reply:
It's overridden in tests to make sure a test servlet is used instead of the real Nest API:

https://github.com/openhab/openhab2-addons/blob/3275662f6ba5a9a094108d928725ec14e54dfdcf/addons/binding/org.openhab.binding.nest.test/src/test/java/org/openhab/binding/nest/test/NestTestBridgeHandler.java#L33-L36